frontend/config/stakingPrograms/optimism.ts

```diff
@@ -0,0 +1,101 @@
+import { Contract as MulticallContract } from 'ethers-multicall';
+
+import { STAKING_TOKEN_PROXY_ABI } from '@/abis/stakingTokenProxy';
+import { AgentType } from '@/enums/Agent';
+import { EvmChainId } from '@/enums/Chain';
+import {
+  OPTIMISM_STAKING_PROGRAM_IDS,
+  OptimismStakingProgramId,
+} from '@/enums/StakingProgram';
+import { TokenSymbol } from '@/enums/Token';
+import { Address } from '@/types/Address';
+
+import { OPTIMISM_STAKING_PROGRAMS_ACTIVITY_CHECKERS } from '../activityCheckers';
+import { StakingProgramConfig } from '.';
+
+export const OPTIMISM_STAKING_PROGRAMS_CONTRACT_ADDRESSES: Record<
+  OptimismStakingProgramId,
+  Address
+> = {
+  [OPTIMISM_STAKING_PROGRAM_IDS.OptimusAlpha1]:
+    '0x88996bbdE7f982D93214881756840cE2c77C4992',
```

Comment:
this contract is no longer in use, please confirm with Pia we should have it in Pearl 
<img width="550" alt="image" src="https://github.com/user-attachments/assets/bb9e4657-6394-46f6-8de7-1902d6eae944" />